Documentation/proposals/202309-controller-id.md

```diff
@@ -0,0 +1,65 @@
+# Controller-ID Proposal
+
+## Owners:
+* [danielmellado](https://github.com/danielmellado)
+
+## Related Tickets:
+* [#4281](https://github.com/prometheus-operator/prometheus-operator/issues/4281)
+* [#4498](https://github.com/prometheus-operator/prometheus-operator/pull/4498)
+
+## Other Docs:
+* n/a
+
+### Introduction
+
+This proposal aims to implement a solution to support multiple cluster-level
+Prometheus instances running concurrently without conflicting over the same CRD.
+This issue can significantly impact use cases where multiple Prometheus operator
+instances run in parallel.
+
+## Why
+
+Currently, we encounter issues when users or different operators deploy Prometheus
+alongside Prometheus Operator's instance. Even though User-Managed Workloads
+(UWM) exists, some operators or users may deploy Prometheus directly from
+manifests, Helm charts, or other rogue operators, leading to the creation of
+'rogue' Prometheus instances alongside the operator's instance.
```

Comment:
I don't think that Prometheus instances deployed without the operator are an issue. I would simplify the paragraph saying that the core issue is when different users deploy different instances of the Prometheus operator which try to reconcile the same resources.